lms/djangoapps/mobile_api/video_outlines/tests.py

```diff
@@ -410,6 +428,76 @@ class TestVideoSummaryList(
     """
     REVERSE_INFO = {'name': 'video-summary-list', 'params': ['course_id']}
 
+    def test_no_mobile_low_profile(self):
+        """Tests VideoSummaryList when there is no mobile_low profile"""
+        self.login_and_enroll()
+        edx_video_id = "testing_mobile_high"
+        api.create_video({
+            'edx_video_id': edx_video_id,
+            'status': 'test',
+            'client_video_id': u"test video omega \u03a9",
+            'duration': 12,
+            'courses': [unicode(self.course.id)],
+            'encoded_videos': [
+                {
+                    'profile': 'youtube',
+                    'url': 'xyz123',
+                    'file_size': 2222,
+                    'bitrate': 4444
+                },
+                {
+                    'profile': 'mobile_high',
+                    'url': self.video_url_high,
+                    'file_size': 111,
+                    'bitrate': 333
+                },
+
+            ]})
+        ItemFactory.create(
+            parent=self.other_unit,
+            category="video",
+            display_name=u"testing mobile high video",
+            edx_video_id=edx_video_id,
+        )
+        course_outline = self.api_response().data
+
+        expected_output = {
+            'category': u'video',
+            'video_thumbnail_url': None,
+            'language': u'en',
+            'name': u'testing mobile high video',
+            'video_url': u'http://val.edx.org/val/video_high.mp4',
+            'duration': 12.0,
+            'transcripts': {
+                'en': 'http://testserver/api/mobile/v0.5/video_outlines/transcripts/{}/testing_mobile_high_video/en'.format(self.course.id)  # pylint: disable=line-too-long
+            },
+            'encoded_videos': {
+                'duration': 12.0,
+                'profiles': {
+                    u'mobile_high': {
+                        'url': u'http://val.edx.org/val/video_high.mp4',
+                        'file_size': 111
+                    },
+                    u'youtube': {
+                        'url': u'xyz123',
+                        'file_size': 2222
+                    }
+                }
+            },
+            'size': 111
+        }
+        course_outline[0]['summary'].pop("id", None)
+        self.assertEqual(course_outline[0]['summary'], expected_output)
+
+    def test_video_profile_list(self):
```

Comment:
This should be in a separate test class in mobile_api/tests.py. There should also be more cases.